shell/browser/extensions/extension_tab_util.h

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SHELL_BROWSER_EXTENSIONS_EXTENSION_TAB_UTIL_H_
+#define SHELL_BROWSER_EXTENSIONS_EXTENSION_TAB_UTIL_H_
+
+#include <memory>
+#include <string>
+#include <vector>
```

Comment:
```suggestion
#include <memory>
```

Unused in this file.